src/include/miopen/conv/invokers/impl_gemm_dynamic.hpp

```diff
@@ -238,29 +238,6 @@ ComputeDynamicIGemmForwardKernelArgs<solver::TunableImplicitGemmGTCDynamic_t>(
     return opArgs;
 }
 
-inline float CallImplGemmDynamicForward(const miopen::Handle& handle,
-                                        ConstData_t src,
-                                        Data_t dst,
-                                        ConstData_t wei,
-                                        const std::vector<OpKernelArg>& opShapeArgs,
-                                        const std::vector<KernelInvoke>& kernels)
-{
-    float elapsed = 0.0f;
-    auto kernel   = kernels[0];
-    std::vector<OpKernelArg> opArgs;
-    opArgs.emplace_back(src);
-    opArgs.emplace_back(wei);
-    opArgs.emplace_back(dst);
-
-    opArgs.insert(opArgs.end(), opShapeArgs.begin(), opShapeArgs.end());
```

Comment:
note: I use `std::transform` with `back_inserter` which should be faster than `insert`.

Reply:
clear.